src/dmd/e2ir.d

```diff
@@ -1720,8 +1720,6 @@ elem *toElem(Expression e, IRState *irs)
                 }
                 else
                 {
-                    d_uns64 elemsize = sd.size(ne.loc);
```

Comment:
Calling `size` _might_ produce an error.

Reply:
Error should be caught by the front-end here:
https://github.com/dlang/dmd/blob/74285e41e670bf7e2ae7d83bc9b1617e12f93a65/src/dmd/expressionsem.d#L2273-L2276